webapp/src/components/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,208 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  Menu,
+  Container,
+  Card,
+  Pagination,
+  Loader,
+  Dimmer,
+  Label
+} from 'semantic-ui-react'
+import AddressLink from 'components/AddressLink'
+import Navbar from 'components/Navbar'
+import Publication from 'components/MarketplacePage/Publication'
+import Parcel from './Parcel'
+import Contribution from './Contribution'
+import { publicationType } from 'components/types'
+
+import { TABS, buildUrl } from './utils'
+
+import './ProfilePage.css'
+
+export default class ProfilePage extends React.PureComponent {
+  static propTypes = {
+    publications: PropTypes.arrayOf(publicationType),
+    page: PropTypes.number.isRequired,
+    pages: PropTypes.number.isRequired,
+    sortBy: PropTypes.string.isRequired,
+    sortOrder: PropTypes.string.isRequired,
+    onNavigate: PropTypes.func.isRequired,
+    onConnect: PropTypes.func.isRequired,
+    onFetchPublications: PropTypes.func.isRequired
+  }
+
+  componentWillMount() {
+    const { onConnect, onFetchAddress } = this.props
+    onConnect()
+    onFetchAddress()
+  }
+
+  handlePageChange = (event, data) => {
+    const { address, tab, onNavigate } = this.props
+    const url = buildUrl({
+      page: data.activePage,
+      address,
+      tab
+    })
+    onNavigate(url)
+  }
+
+  renderLoading() {
+    return (
+      <Dimmer active inverted>
```

Comment:
I think we can safely remove the `Dimmer` here, because the loading is inside the Tab content so it looks the same (and I actually like it that way, dimming can be a bit annoying)

Reply:
This is a pattern enforced by Semantic UI, `<Loader>` has a `display: none;` by default unless it's inside a `<Dimmer>`

Reply:
> Loaders are hidden unless has prop `active` or inside an `Dimmer active`.

my bad 🤦‍♂️ updated!